core-bundle/src/Resources/config/services.yml

```diff
@@ -145,6 +145,8 @@ services:
 
     contao.csrf.token_storage:
         class: Contao\CoreBundle\Csrf\MemoryTokenStorage
+        tags:
+            - { name: kernel.reset, method: reset }
```

Comment:
You also have to adjust the test in the `ContaoCoreExtensionTest` class (for all changed services).